<?xml version="1.0" encoding="utf-8"?>
<launch>
    <param name ="/use_sim_time" value="true"/>
    <param name = "result_file" type="string" value="$(find cam_lidar_calib)/result/C_T_L_nerian_left_r_faccal.txt"/>

    <param name = "cloud1_input_topic" type="string" value="/nerian_stereo/point_cloud"/>
    <param name = "cloud2_input_topic" type="string" value="/nerian_stereo/left_image/velodyne_out_cloud"/>
    <param name = "cloud2_output_topic" type="string" value="/velodyne_points_transformed"/>

    <node  pkg="cam_lidar_calib" name="cloudProjector" type="lidar_stereo_pcd_proj_node" output="screen">
    </node>
</launch>
